src/com/spikes2212/Cameras/CamerasHandler.java

```diff
@@ -12,15 +12,22 @@
 	private ArrayList<CameraController> cameras;
 	private CameraController currentCamera = null;
 	private Image image;
+	private int currentIndex;
 	public static int fps = 30;

```

Comment:
make this not static, so different camera handlers can have different fps values